src/unread/UnreadCards.js

```diff
@@ -107,5 +107,5 @@ export default connect((state: GlobalState) => ({
   conversations: getUnreadConversations(state),
   presences: getPresence(state),
   usersByEmail: getAllUsersByEmail(state),
-  unreadStreamsAndTopics: getUnreadStreamsAndTopicsSansMuted(state),
+  unreadStreamsAndTopics: [], // getUnreadStreamsAndTopicsSansMuted(state),
```

Comment:
WIP?

Reply:
Oh! ya. Thanks @borisyankov 
I used it for testing purpose 😅 
My mistake forgot to remove it before commiting. Will take care of it next time 👍